WordPress/Classes/Extensions/UIImageView+SiteIcon.swift

```diff
@@ -92,13 +94,9 @@ extension UIImageView {
             case .failure(let error):
                 if case .requestCancelled = (error as? AFIError) {
                     // Do not log intentionally cancelled requests as errors.
-                } else if case let Alamofire.AFError.responseValidationFailed(reason) = error,
-                          case let Alamofire.AFError.ResponseValidationFailureReason.unacceptableStatusCode(code) = reason,
-                          code == 404 {
-                    // Do not log 404 errors since they are expected for site icons
-                }
-                else {
```

Comment:
Removing this conditional branch means if we get a 404, it will be logged.

Did you do this intentionally to help diagnosing the root cause of the issue?

Reply:
I may be wrong about my interpretation.  Ping @bjtitus since I think you know the conext of this.

The reason why I think the error was not being logged at all is that we were logging it as an error to Sentry, which is quite more "aggressive" than regular logging.

In terms of Groundskeeping this can come across as a crash and raise alarms.

With my change we're logging both 404 and 400 errors locally so we can figure out eventually the root cause behind the downloads failing - if the site icon is fine, and the URL is correct, we should get none of these errors AFAIK.  My theory is we're breaking some icons with the URL optimization we're doing for photon, so logging cases in which we're receiving a 404 is, IMHO, desired.

In practical terms, when I look at the next log file, I would like to know of any errors here that are not caused by the user cancelling the download.

Does that make sense?

Reply:
Makes total sense to me 👍 I think logging everything makes total sense. I mainly asked because that error was originally ignored and I wondered why 👍

Reply:
That interpretation is correct. The original intent is maintained if the errors are just logged locally.

Reply:
Thanks @bjtitus 👍